cli/cli.go

```diff
@@ -179,7 +179,11 @@ func SetupCLI(args []string) error {
 					Destination: &runOptions.rebootExitCode,
 					Usage: "Return exit code 4 if a manual reboot " +
 						"is required after the Artifact installation.",
-				},
+				},&cli.StringFlag{
+			Name:        "key-passphrase",
+			Usage:       "Passphrase for decrypting an encrypted private key",
+			Value:       "",
+			Destination: &runOptions.keyPassphrase},
```

Comment:
Note that passing in a passphrase on the command line is not secure, it can be seen by all users on the system through the process table. It should either be passed in via a file or via stdin. May I suggest a filename argument, where `-` means opening stdin?